app/code/Magento/MediaGallery/Test/Unit/Model/DataExtractorTest.php

```diff
@@ -54,22 +53,41 @@ public function testExtractData(string $class, $interfaceClass, array $expectedD
                 'data' => $data,
             ]
         );
-        $receivedData = $this->dataExtractor->extract($model, $interfaceClass);
-        $this->checkValues($expectedData, $receivedData, $model);
+        if ($interfaceClass) {
```

Comment:
I don't see the reason to change the test logic after adding a `size` field. Only data provider should be changed. What is the reason for this change?

Reply:
I'm trying to improve the Unit test for this line https://github.com/magento/magento2/blob/1a77b70fc59ba3d0d1cae76f40be567baa75041e/app/code/Magento/MediaGallery/Test/Unit/Model/DataExtractorTest.php#L72